### PR TITLE
JAVACLIENT-108 : set read timeout

### PR DESCRIPTION
**Pull request by funshodavid**

### CODE REVIEW

**Comment by pgmillon** [comment]
Maybe it could be interesting to offer 2 different variables ?

**Comment by funshodavid** [comment]
Or @pgmillon, what about keeping it that way so that it's the default behaviour, but exposing a method for read timeout only ?

**Comment by pgmillon** [comment]
Agreed. I was thinking of read/connectTimeout but it could be worse for readability.

File: nuxeo-java-client/src/main/java/org/nuxeo/client/api/Client.java [hunk 1]
```diff
@@ -59,6 +59,11 @@
     NuxeoClient timeout(long timeout);
 
     /**
+     * @since 2.3
+     */
+    NuxeoClient setReadTimeout(long timeout);
```
Comment (ataillefer) [inline nuxeo-java-client/src/main/java/org/nuxeo/client/api/Client.java]: Could be called `readTimeout()` for consistency with `timeout()`

**Comment by ataillefer** [comment]
:+1:

**Comment by pgmillon** [comment]
👍

**Pull request merged**